tests/e2e/env/setup.sh

```diff
@@ -54,8 +54,8 @@ fi
 step "Setting up server containers"
 redirect_output local/bin/docker-setup.sh
 
-step "Configuring server with stripe account"
-redirect_output $SERVER_PATH/local/bin/link-account.sh $BLOG_ID $E2E_WCPAY_STRIPE_ACCOUNT_ID
+step "Configuring server with stripe account (test account, skip sync)"
```

Comment:
> (test account, skip sync)

Didn't get this comment. Should it be `(stripe account, mode, skip sync)`?

Besides that, looks good.

Reply:
Hm, it was intended to explain that we use a test account and skip sync, but not the order of the flags. Since it's was confusing already, let's just revert the ` (test account, skip sync)` part =)